src/lib/components/ProjectsPage/ProjectsTable.tsx

```diff
@@ -56,9 +58,22 @@ export const ProjectsTable = ({ projectType }: IProjectPageProps) => {
     setActiveFilters({ ...activeFilterFields });
   }, [watchedFields]);
 
+  const filterBySearch = (row: any) => {
+    if (searchValue.length) {
+      return searchBy
+        .map((field) => {
+          return row[field].toString().includes(searchValue);
+        })
+        .some((row) => row === true);
+    } else {
+      return true;
+    }
+  };
+
   const rows: IProjectDoc[] = data
     .filter((p) => p.projectType === projectType)
-    .filter((r) => filterByFilterPanel(r, watchedFields as any)); //TODO: fix that type assertion
+    .filter((r) => filterByFilterPanel(r, watchedFields as any)) //TODO: fix that type assertion
+    .filter(filterBySearch);
```

Comment:
```suggestion
    .filter((r) => filterBySearch(r, searchBy as any, searchValue)); // TODO: fix that type assertion
```

Reply:
i was sruggling how to pass searchValue and the answer was right here...

ok ill change it!